cmd/ci-operator-prowgen/main_test.go

```diff
@@ -285,6 +288,7 @@ func TestGeneratePodSpecTemplate(t *testing.T) {
 
 func TestGeneratePresubmitForTest(t *testing.T) {
 	newTrue := true
+	standardJobLabels := map[string]string{jobconfig.ProwJobLabelGenerated: jobconfig.Generated}
```

Comment:
Can we not use the imports here just so we test the actual values and someone can't break us by helpful refactors?

Reply:
Sure.